src/cf_gen.cc

```diff
@@ -97,6 +97,9 @@ class Entry
     LineList cfgLines; ///< between CONFIG_START and CONFIG_END
     int array_flag = 0; ///< TYPE is a raw array[] declaration
 
+    /// whether this is a "REPETITIONS: update" entry
```

Comment:
IMO calling the option **REPEATS** would help avoid problems from developer typos.

Reply:
The cardinality of the set of people affected by this is depressingly low.
Not worth discussing